### DISCUSSION TITLE
Modal plot_mode_3d() animation

### DISCUSSION THREAD

**Discussion by fmegally** [petrobras/ross#1129] [Q&A]
Hi

I am trying to use `animation=True` argument for `plot_mode_3d()`  to better visualize "mixed whirl" condition but it doesn't seem to be working.  The plot shows but it exactly like if the argument was False

Are there any library requirements or plotly settings that i need to change for it work properly ?

![image](https://github.com/user-attachments/assets/d46d68e8-0dcb-43b2-8d9a-8333e806354b)

**Comment by jguarato** [comment]
Hi @fmegally,

The `animation` argument currently only works for torsional modes.
